### ISSUE TITLE
"behave like" is not working

### ISSUE THREAD

**Issue by GoogleCodeExporter** [etorreborre/specs#51]
```
What steps will reproduce the problem?
1. try to run example program "stackSpec.scala" (referenced on project
home). It does not work.

What is the expected output? What do you see instead?
The execution report for reused example is printed two times, 
but example is executed only once.

What version of the product are you using? On what operating system?
1.4.1

Please provide any additional information below.
Seems like Example.executed flag is preventing the example to be executed 
second time?


```

Original issue reported on code.google.com by `udovenk...@gmail.com` on 23 Jan 2009 at 12:48

**Comment by GoogleCodeExporter** [comment]
```
Thanks for noticing this, this is fixed in version 1.4.3-SNAPSHOT.
```

Original comment by `etorrebo...@gmail.com` on 23 Jan 2009 at 3:56
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `etorrebo...@gmail.com` on 25 Feb 2009 at 9:37
- Changed state: **Verified**
